Client/src/components/rightBar.tsx

```diff
@@ -0,0 +1,136 @@
+import '../css/rightBar.css';
+import ShopSVG from './svg/ShopSVG';
+import { animal, minionType } from '../utils/types';
+import { Squirrel, Badger, Hare, Deer, Koala, Bear } from './svg/animalsSVG';
+import { useState } from 'react';
+import Shop from './shop';
+import CloseCross from './svg/closeCross';
+
+function storeButtonHover(isOnHover: boolean) {
+  document.querySelectorAll('.store-button-yellow').forEach((svgEl) => {
+    isOnHover
+      ? ((svgEl as unknown as HTMLElement).style.fill = `var(--red)`)
+      : ((svgEl as unknown as HTMLElement).style.fill = `var(--yellow)`);
+  });
+  document.querySelectorAll('.store-button-red').forEach((svgEl) => {
+    isOnHover
+      ? ((svgEl as unknown as HTMLElement).style.fill = `var(--yellow)`)
+      : ((svgEl as unknown as HTMLElement).style.fill = `var(--red)`);
+  });
+}
+
+function crossButtonHover(isOnHover: boolean) {
+  document.querySelectorAll('.close-cross-green').forEach((svgEl) => {
+    isOnHover
+      ? ((svgEl as unknown as HTMLElement).style.fill = `#96B0A2`)
```

Comment:
What is this colour? If you're using it, please add it to the global variables to keep consistency

Reply:
Done